src/main/scala/org/broadinstitute/hail/variant/Variant.scala

```diff
@@ -282,8 +284,8 @@ case class Variant(contig: String,
   def inXNonPar: Boolean = isX && !inXParPos
   def inYNonPar: Boolean = isY && !inYParPos
```

Comment:
`X` is not an adjective like autosomal or mitochondrial, so I'm bothered by `isX`.  I'd prefer `inX` as short for "in the X chromosome" though I know it's more common to say `onX`.  I used `inXPar` as short for "in the non-pseudoautosomal part of X".